src/index.ts

```diff
@@ -32,6 +32,7 @@ import { PermissionRoles, PermissionTypes } from './labkey/security/constants';
 // modules
 import * as ActionURL from './labkey/ActionURL';
 import * as Ajax from './labkey/Ajax';
+import { RequestOptions } from './labkey/Ajax';
```

Comment:
It's fine to export these but they should already (and are?) be exported under their namespace. Are you not seeing this?

<img width="555" alt="image" src="https://github.com/user-attachments/assets/9b1a3555-f9c9-4c38-96ca-1abdaacc2ca6" />

<img width="580" alt="image" src="https://github.com/user-attachments/assets/e0763c40-3b21-44d6-8b91-3e5096569b80" />

Reply:
Intellij can't figure that out on its own, so it was always trying to import from the dist folder. I'll revert these changes. I do have other changes to Command that will help us improve types downstream so this PR is still relevant.